pkg/astvalidation/definition_validation_test.go

```diff
@@ -0,0 +1,27 @@
+package astvalidation
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/jensneuse/graphql-go-tools/pkg/astparser"
+	"github.com/jensneuse/graphql-go-tools/pkg/asttransform"
+)
+
+func runDefinitionValidation(t *testing.T, definitionInput string, expectation ValidationState, rules ...Rule) {
+	definition, report := astparser.ParseGraphqlDocumentString(definitionInput)
+	require.False(t, report.HasErrors())
+
+	err := asttransform.MergeDefinitionWithBaseSchema(&definition)
```

Comment:
should we do the same on validation stage? or it will be done before running validator?

Reply:
this will be done before validation runs or at least it doesn't make sense to validate the document without merging he base schema before